docs/content/installation/integrations/app-protect-waf-v5/installation.md

```diff
@@ -0,0 +1,241 @@
+
+To upload the image, run the following command. If you're using a custom tag, add `TAG=your-tag` to the end of the command. Replace `<my-docker-registry>` with your private registry's path.
+
+```shell
+make push PREFIX=<my-docker-registry>/nginx-plus-ingress
+```
+
+To upload the WAF config manager and enforcer images run the following commands:
+
+```shell
+docker push <my-docker-registry>/waf-config-mgr:<your-tag>
+```
+
+```shell
+docker push <my-docker-registry>/waf-enforcer:<your-tag>
+```
+
+---
+
+## Set up role-based access control (RBAC) {#set-up-rbac}
+
+{{< include "rbac/set-up-rbac.md" >}}
+
+---
+
+
+{{< include "installation/create-custom-resources.md" >}}
+
+
+## Deploy NGINX Ingress Controller {#deploy-ingress-controller}
+
+{{< include "installation/deploy-controller.md" >}}
+
+{{< note >}} NGINX Ingress Controller with the AppProtect WAF v5 module works only with policy bundles. You need to modify the Deployment or DaemonSet file to include volumes, volume mounts and two WAF 5 docker images: `waf-config-mgr` and `waf-enforcer`.
+
+NGINX Ingress Controller **requires** the volume mount path to be `/etc/app_protect/bundles`. {{< /note >}}
+
+Add a `volumes` section to deployment template spec:
+
+```yaml
+...
+volumes:
+- name: <volume_name>
+persistentVolumeClaim:
+    claimName: <claim_name>
+...
+```
+
+Add volume mounts to the `containers` section:
+
+```yaml
+...
+volumeMounts:
+- name: <volume_mount_name>
+    mountPath: /etc/app_protect/bundles
+...
+```
+
+
+Add `waf-config-mgr` image to the `containers` section:
+
+```yaml
+...
+- name: waf-config-mgr
+  image: private-registry.nginx.com/nap/waf-config-mgr:<version-tag>
+  imagePullPolicy: IfNotPresent
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - all
+  volumeMounts:
+    - name: app-protect-bd-config
+      mountPath: /opt/app_protect/bd_config
+    - name: app-protect-config
+      mountPath: /opt/app_protect/config
+    - name: app-protect-bundles
+      mountPath: /etc/app_protect/bundles
+...
+```
+
+
+Add `waf-enforcer` image to the `containers` section:
+
+```yaml
+...
+- name: waf-enforcer
+  image: private-registry.nginx.com/nap/waf-enforcer:<version-tag>
+  imagePullPolicy: IfNotPresent
+  env:
+    - name: ENFORCER_PORT
+      value: "50000"
+  volumeMounts:
+    - name: app-protect-bd-config
+      mountPath: /opt/app_protect/bd_config
+...
+```
+
+
+### Using a Deployment
+
+{{< include "installation/manifests/deployment.md" >}}
+
+### Using a DaemonSet
+
+{{< include "installation/manifests/daemonset.md" >}}
+
+---
+
+## Enable NGINX App Protect WAF module
+
+To enable the NGINX App Protect DoS Module:
+
+- Add the `enable-app-protect` [command-line argument]({{< relref "configuration/global-configuration/command-line-arguments.md#cmdoption-enable-app-protect" >}}) to your Deployment or DaemonSet file.
+
```

Comment:
Do we need a reference here for the equivalent Helm configuration?

Reply:
fixed